src/omnibar/omnibar.spec.ts

```diff
@@ -1284,14 +1286,79 @@ describe('Omnibar', () => {
 
       getTokenFake = () => Promise.resolve(testTokenWithNotificationEntitlement);
 
-      loadOmnibar();
+      loadOmnibarWithNotifications();
 
       fireMessageEvent({
         messageType: 'get-token',
         tokenRequestId: 123
       });
     });
 
+    describe('handle notifications per svcid', () => {
```

Comment:
A nested `describe` should follow the parent `describe` logically as if it were a sentence, so since the outer describe is "Omnibar," maybe this describe should be "when connecting to notifications," then describe the action in the `it` blocks.